gradle-conjure/src/main/java/com/palantir/gradle/conjure/GenerateNpmRcTask.java

```diff
@@ -86,8 +86,8 @@ public final void createNpmrc() throws IOException, InterruptedException {
                         tokenFromCreds(registryUri, username, password).token())
                 : "";
 
-        String scopeRegistry = scope.map(s -> s + ":").orElse("");
-        String npmRcContents = scopeRegistry + "registry=" + registryUri + tokenString;
+        String scopeRegistry = scope.map(s -> "@" + s + ":").orElse("");
```

Comment:
the `@` sign was stripped from earlier and is not used for making request but it should be in the npmrc